v1_21/src/main/java/net/countercraft/movecraft/support/v1_21/ISmoothTeleport.java

```diff
@@ -10,11 +10,11 @@ public class ISmoothTeleport extends SmoothTeleport {
     public void teleport(@NotNull Player player, @NotNull Location location) {
         player.teleport(
                 location,
-                TeleportFlag.Relative.X,
-                TeleportFlag.Relative.Y,
-                TeleportFlag.Relative.Z,
-                TeleportFlag.Relative.PITCH,
-                TeleportFlag.Relative.YAW,
+                TeleportFlag.Relative.VELOCITY_X,//x
+                TeleportFlag.Relative.VELOCITY_Y,//y
+                TeleportFlag.Relative.VELOCITY_Z,//z
+                TeleportFlag.Relative.VELOCITY_ROTATION,//pitch
+                TeleportFlag.Relative.VELOCITY_ROTATION,//yaw
```

Comment:
Is this confirmed to be the same behavior as before?

Reply:
Yes. The flags did have the same as the original ones. 

Here is an excerpt from the TeleportFlag class:
```java
   @Deprecated(
            since = "1.21.3",
            forRemoval = true
        )
        public static final Relative X = VELOCITY_X;
```